snmp/tests/test_e2e_core_metadata.py

```diff
@@ -31,7 +31,7 @@ def assert_metadata_events(aggregator, events):
 def assert_device_metadata(aggregator, device_metadata):
     events = get_events(aggregator)
 
-    assert len(events) == 1
+    assert len(events) >= 1
```

Comment:
For reviewers:
this allows to accept devices with more than 99 interfaces (an event allows for 1 device + 99 interfaces)